tests/test-definitions.txt

```diff
@@ -87,7 +87,7 @@ recovery_cluster priority=2000 cluster buckets=8 -- --test search
 shell_server_aql priority=1000 cluster buckets=16 -- --dumpAgencyOnError true
 shell_client priority=500 cluster buckets=5 -- --dumpAgencyOnError true
 shell_client_transaction priority=500 cluster parallelity=5 buckets=5 -- --dumpAgencyOnError true
-#shell_client_replication2_recovery priority=500 cluster buckets=5 -- --dumpAgencyOnError true
+shell_client_replication2_recovery priority=500 cluster buckets=5 -- --dumpAgencyOnError true
```

Comment:
This is a potentially unrelated change. Is it intentional?

Reply:
Yes, this test was blocked by this issue. At least @dothebart  asked me to uncomment it.

Reply:
Thanks for clarifying!